### PR TITLE
Add option to show only the image without the description

### PR DESCRIPTION
**Pull request by caraquista**
Bumps [acorn](https://github.com/acornjs/acorn) from 6.1.1 to 6.4.1.
- [Release notes](https://github.com/acornjs/acorn/releases)
- [Commits](https://github.com/acornjs/acorn/compare/6.1.1...6.4.1)

Signed-off-by: dependabot[bot] <support@github.com>

### CODE REVIEW

File: src/components/CardMedia.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-declare const CardMedia: ({ data, cardSize, autoPlay }: {
+declare const CardMedia: ({ data, cardSize, autoPlay, isShowDescription }: {
     data: any;
     cardSize: any;
     autoPlay: any;
+    isShowDescription: any;
```
Comment (winhtaikaung) [inline src/components/CardMedia.d.ts]: Shall we make it as boolean ? :thinking:

File: src/demo/dist/index.html [hunk 2]
```diff
@@ -43,7 +43,7 @@
     
     
 
-    <title>React Tiny Link(Link Preview)</title>
+    <title>React Tiny Link(Link Preview) DANIEL</title>
```
Comment (winhtaikaung) [inline src/demo/dist/index.html]: Can you please remove ` DANIEL` :pray:

**Review by winhtaikaung — CHANGES_REQUESTED** [review]
It would be great if you could provide screenshot for your changes @caraquista

File: src/components/CardMedia.d.ts (hunk 1 above)
Comment (caraquista) [reply src/components/CardMedia.d.ts]: Done

**Comment by caraquista** [comment]
> It would be great if you could provide screenshot for your changes @caraquista

Screenshot from 2020-03-24 12-56-31

**Comment by winhtaikaung** [comment]
> > It would be great if you could provide screenshot for your changes @caraquista
> 
> Screenshot from 2020-03-24 12-56-31

Hi @caraquista, I think its wrong picture, could you please update with the picture of component

**Comment by caraquista** [comment]
> > > It would be great if you could provide screenshot for your changes @caraquista
> > 
> > 
> > Screenshot from 2020-03-24 12-56-31
> 
> Hi @caraquista, I think its wrong picture, could you please update with the picture of component

That picture is the component in action.... what image do you want??

**Comment by erfoon** [comment]
Isn't this feature merged? Seems to be so necessary, we have `showGraphics` but not `showDescription`?

**Pull request closed without merging**